group/edwards25519/fe_test.go

```diff
@@ -0,0 +1,64 @@
+package edwards25519
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.dedis.ch/kyber/v3/xof/blake2xb"
+)
+
+func Test_feToBnZero(t *testing.T) {
+	var f0 fieldElement
+	feZero(&f0)
+
+	b0 := big.NewInt(0)
+	feToBn(b0, &f0)
+
+	assert.Equal(t, big.NewInt(0).Cmp(b0), 0)
+}
+
+func Test_feBnConversionRandom(t *testing.T) {
```

Comment:
I agree that property testing is a good way to test the conversion, but maybe you can use other properties that involve multiplication (e.g., distributive) to make the test stronger.